modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -30,6 +40,11 @@ spec:
 
 . Configure the `KubeDescheduler` object with the `LongLifecycle` profile and enable background evictions for improved VM eviction stability during live migration:
 +
+[NOTE]
+====
+When the eviction annotation is set on VMs, the `LongLifecycle` profile is sufficient for VM evictions. You do not need to enable `EvictPodsWithLocalStorage` or `EvictPodsWithPVC`.
```

Comment:
Since it's only supported to do LongLifecycle, recommend:

```suggestion
When the eviction annotation is set on VMs, the `LongLifecycle` profile is sufficient for VM evictions. Do not enable `EvictPodsWithLocalStorage` or `EvictPodsWithPVC`.
```